src/Plugin/Satis/UpdateAndBuildJob.php

```diff
@@ -9,20 +9,41 @@
 
 namespace Terramar\Packages\Plugin\Satis;
 
+use Doctrine\ORM\EntityManager;
 use Symfony\Component\Process\PhpExecutableFinder;
 use Symfony\Component\Process\ProcessBuilder;
+use Terramar\Packages\Entity\Package;
 use Terramar\Packages\Job\ContainerAwareJob;
 
 class UpdateAndBuildJob extends ContainerAwareJob
 {
+    /**
+     * @return EntityManager
+     */
+    private function getEntityManager()
+    {
+        return $this->getContainer()->get('doctrine.orm.entity_manager');
+    }
+
     public function run($args)
     {
+        /** @var Package $package */
+        $package = $this->getEntityManager()->getRepository(Package::class)->find($args['package_id']);
+
         $finder = new PhpExecutableFinder();
-        $builder = new ProcessBuilder(array('bin/console', 'satis:update', '--build', '--skip-errors'));
+        $builder = new ProcessBuilder([
+            'bin/console',
+            'satis:build',
+            '--repository-url',
+            $package->getSshUrl(),
+        ]);
+
         $builder->setEnv('HOME', $this->getContainer()->getParameter('app.root_dir'));
         $builder->setPrefix($finder->find());
         $builder->setTimeout(null);
 
+        echo $builder->getProcess()->getCommandLine() . "\n";
```

Comment:
This is probably valuable to have, so I think we should keep it (in case there was a question in the first place ;)

Reply:
It's not always great to leave `echo` in, and I should probably have found something to log it cleanly, but it certainly works for it's intended purpose.